README.md

```diff
@@ -51,6 +51,10 @@ This object represents the current state of a thermostat.
 | `manual_temperature` | integer | If the regulation mode is set to manual mode, the thermostat will target this temperature. |
 | `comfort_temperature` | integer | If the regulation mode is set to comfort mode, the thermostat will target this temperature. |
 | `comfort_end_time` | datetime | If the regulation mode is set to comfort mode, it will end at this time. |
+| `vacation_mode` | boolean | If true then the thermostat is set to `vacation_temperature` from `vacation_begin_time` to `vacation_end_time`. If false, then no vacation is in progress or scheduled. |
```

Comment:
`vacation_mode` is a confusing name since we also have `regulation_mode`. The API calls this `VacationEnabled` which is more accurate.

This boolean does not indicate *whether we are currently in the vacation regulation mode*. It indicates *whether a vacation is currently scheduled*, regardless of whether it begins in the past or future.

Reply:
(I tried to update this documentation to make it a bit clearer.)

Reply:
If you want to change it to `vacation_enabled` thats fine with me, makes more sense :-)

Reply:
Nah, that's a backwards incompatible change so I figure we should leave it for the next major version bump.